From: Marco Martin <notmart@gmail.com>
Date: Mon, 9 Jan 2017 16:29:32 +0100
Subject: portait prop is not relevant when there is no text

this fixes a binding loop on this property
when there is no label

BUG:374815
Change-Id: I71616a6c86cd4acdaff47f141ed5c676358db496
---
 src/declarativeimports/plasmacomponents/qml/TabButton.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/declarativeimports/plasmacomponents/qml/TabButton.qml b/src/declarativeimports/plasmacomponents/qml/TabButton.qml
index 748354582..ff1af52ad 100644
--- a/src/declarativeimports/plasmacomponents/qml/TabButton.qml
+++ b/src/declarativeimports/plasmacomponents/qml/TabButton.qml
@@ -122,7 +122,7 @@ Item {
 
         property Item tabBar: Utils.findParent(root, "currentTab")
         property Item tabGroup: Utils.findParent(tab, "currentTab")
-        property bool portrait: (root != undefined) && (label != undefined) && root.height >= label.paintedHeight + units.iconSizes.small
+        property bool portrait: (root != undefined) && (label != undefined) &&  label.text != "" && root.height >= label.paintedHeight + units.iconSizes.small
 
         function click() {
             root.clicked()
